cmd/fluent-bit/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.13.6-alpine3.11 as buildergo
```

Comment:
can we rename  dir `cmd/fluent-bit` to  `cmd/fluent-bit-watcher` ?
Also pls make corresponding changes to the content of Dockerfile